app/assets/fonts/Canterbury/Canterbury.svg

```diff
@@ -0,0 +1,635 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Please put all the font files in Amazon AWS S3. Doing version control for large static assets will make git look very messy.